### PR TITLE
use `@waku/sdk`

### PR DESCRIPTION
**Pull request by felicio**
awaits https://github.com/status-im/status-web/pull/507

### CODE REVIEW

**Review by jkbktl — APPROVED** [review]
✅ locally tested

File: packages/DApp/src/providers/waku/provider.tsx [hunk 1]
```diff
@@ -24,7 +23,9 @@ export function WakuProvider({ children }: Props) {
         // @ts-ignore
         emitSelf: true,
         libp2p: {
-          peerDiscovery: [new PeerDiscoveryStaticPeers(peers[config.wakuConfig.environment], { maxPeers: 1 })],
+          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+          // @ts-ignore icorrectly exported types from @libp2p/boostrap#package.json; patched only in @status-im/js for now
+          peerDiscovery: [bootstrap({ list: peers[config.wakuConfig.environment] })],
```
Comment (felicio) [inline packages/DApp/src/providers/waku/provider.tsx:28]: Set `timeout` and `tagTTL`, ref https://libp2p.github.io/js-libp2p/modules/_libp2p_bootstrap.html.

Comment (felicio) [reply packages/DApp/src/providers/waku/provider.tsx]: https://github.com/status-im/community-dapp/pull/102/commits/83c99ed8a72d0fdaf1ea56cc4d6b3d62da68a555

Comment (felicio) [reply packages/DApp/src/providers/waku/provider.tsx]: `tagTTL: Infinity` actually prevented waku connection

**Pull request merged**